packages/crypto/Cargo.toml

```diff
@@ -20,6 +20,7 @@ bench = false
 
 [dependencies]
 k256 = { version = "0.13.1", features = ["ecdsa"] }
+ecdsa = "0.16.2"
```

Comment:
Can you add a comment why it is added here? In the long run we should remove it as a direct dependency.

Reply:
`k256` re-exports `ecdsa`, but in minimal version 0.16.0.
I tried to convinced the maintainer of this crate to bump it to version 0.16.8 (newest) but without success.
Our code in crypto uses functions introduced in 0.16.2. So our code does not compile with `-Zminimal-versions` option. That's why I proposed this change, which is of course temporary, until `k256` re-exports `ecdsa` in version usable for us.
Another approach is to try to use `ecdsa` directly and have this dependency under control.
Yet another approach would be to remove from our code the usage of functions not supported in 0.16.0.
Or we can just skip this change and do not bother with it.

Reply:
Yeah, sure, this all makes sense. All I mean is educate future-you that this is the reason this line is in here. E.g.

```suggestion
# Not used directly but need to bump transitive dependency (https://github.com/CosmWasm/cosmwasm/pull/1899)
ecdsa = "0.16.2"
```

Reply:
Done.